synapse/federation/federation_server.py

```diff
@@ -1013,7 +1013,9 @@ async def on_claim_client_keys(
                 query.append((user_id, device_id, algorithm))
 
         log_kv({"message": "Claiming one time keys.", "user, device pairs": query})
-        results = await self._e2e_keys_handler.claim_local_one_time_keys(query)
+        results = await self._e2e_keys_handler.claim_local_one_time_keys(
+            query, always_include_fallback_keys=False
+        )
```

Comment:
Do we need a unstable federation endpoint here too?

Reply:
See https://github.com/matrix-org/matrix-spec-proposals/pull/3983#discussion_r1175534364

Reply:
I added one since the next PR wants one anyway and it seems clearer to do it here then build on it.